crates/subspace-runtime/src/lib.rs

```diff
@@ -844,13 +842,33 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
     generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 
+pub struct InitializeDomainChainAllowlistUpdate;
+
+impl OnRuntimeUpgrade for InitializeDomainChainAllowlistUpdate {
+    fn on_runtime_upgrade() -> Weight {
```

Comment:
We do need to do this upgrade if its just ValueQuery I believe

Reply:
Should be answered in https://github.com/subspace/subspace/pull/2746/files#r1599818561